components/user/userApi.ts

```diff
@@ -160,6 +160,14 @@ export const userApi = createApi({
         url: `/auth/language/${language}`,
```

Comment:
Can we change the input type from QUserLanguage to something that only stores `language`